src/components/MewSuperButton/MewSuperButton.vue

```diff
@@ -253,36 +285,43 @@ export default {
         classes.push('column-reverse');
       }
       return classes;
-    }, 
+    },
     onBtnClick() {
-      this.active = !this.active;
+      //this.active = !this.active;
     },
     getColor() {
       const colorThemesWhite = ['outline', 'basic'];
       if (colorThemesWhite.indexOf(this.colorTheme) >= 0) {
-        return 'white basic-border';
+        return 'white';
```

Comment:
why did you remove basic border? i added it because of russ new hardware designs